funnel/models/rsvp.py

```diff
@@ -40,23 +38,60 @@ class Rsvp(TimestampMixin, db.Model):
         'state',
         db.CHAR(1),
         StateManager.check_constraint('state', RSVP_STATUS),
-        default=RSVP_STATUS.A,
+        default=RSVP_STATUS.MAYBE,
         nullable=False,
     )
     state = StateManager('_state', RSVP_STATUS, doc="RSVP answer")
 
+    @with_roles(call={'auth'})
```

Comment:
Wrong role. Use `owner`. Using `auth` allows anyone who is logged in to change anyone else's response status.

Reply:
Also specify a grant for `owner` in the relationship.

Reply:
> Also specify a grant for owner in the relationship.

What do you mean by that?